### PR TITLE
Faas 1114/building front ends

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

Building front ends tutorial

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+
+
+## Connecting SCCOS
+
+In the *Commerce provider* section, you need to connect the Storefront to a [Spryker Cloud Commerce OS](/docs/cloud/dev/spryker-cloud-commerce-os/getting-started-with-the-spryker-cloud-commerce-os.html) environment and add [locales](/docs/scos/dev/back-end-development/data-manipulation/datapayload-conversion/multi-language-setup.html) and [currencies](/docs/scos/dev/back-end-development/data-manipulation/datapayload-conversion/multiple-currencies-per-store-configuration.html). Follow the steps below:
+
+1. Select a **SPRYKER ENVIRONMENT** to connect this Storefront to.
+
+2. Select one or more **LOCALES** to be available in this Storefront.
```
Comment (tobi-or-not-tobi) [inline docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md:75]: we also need to select the default locale

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+3. Select one or more **CURRENCIES** to be available in this Storefront.
```
Comment (tobi-or-not-tobi) [inline docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md:77]: we also need to select the default currency

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+4. Select **Create**.
+
+The *List of Applications* page opens with the success message displayed. The application is displayed in the list.
+
+## Publishing the Storefront
```
Comment (tobi-or-not-tobi) [inline docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md:83]: Is it publishing? I thought it was named "install"

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Mobile-first design.
 
 ## Who the users of FeS are?
 
-FeS is designed to be used by devlopers and content managers.
+FeS is designed to be used by developers and content managers.
```
Comment (tobi-or-not-tobi) [inline docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service.md:22]: FES

Comment (tobi-or-not-tobi) [reply docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service.md:22]: @alistairSpryker I think "content managers" is not the right user name for now. We could start this docs with developers for now.

**Review by tobi-or-not-tobi — CHANGES_REQUESTED** [review]

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+title: Launching Storefronts
+description: Learn how to launch Storefronts
+template: concept-topic-template
+---
+
+This tutorial describes how to deploy Storefronts using the Spryker Launchpad.
+
+## Logging into the Launchpad
+
+Using the login details provided by your FeS account owner, log into the [Launchpad](https://launchpad.spryker.com).
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md:11]: @alistairSpryker Please confirm the final version

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+To create a Storefront, do the following:
+
+1. On the *List of Applications* page, select **Create new application**.
+
+2. Enter a descriptive **Name** for the Storefront.
+
+3. For **Type**, select **Spryker VueStorefront**.
+  In future, you'll be able to create other types of applications. For now, let's stick with the Storefront.
+
+  *Code Repository provider*, *Hosting provider*, and *Commerce provider* sections appear.
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md:26]: Are we putting them in this order in 1.0 release?

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+This takes you back to the *Create new Application* page.
+
+3. Select the **ORGANISATION** you've connected.
+
+4. Select the **REPOSITORY** with the VueStorefront application.
+
+5. Select the **BRANCH** you want to deploy.
+
+We recommend mapping branches to Storefronts as follows:
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md:44]: TBD with @alistairSpryker

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+In the *Hosting provider* section, you need to configure how the Storefront will be deployed to [Netlify](https://www.netlify.com/). In future, you'll also be able to use other hosting providers.
+
+To configure the build configuration, do the following:
+
+1. Optional: Enter a **BASE DIRECTORY**.
+  This is the directory from which a deployment is started. If not specified, the root directory of the repository is used.
+
+2. Optional: Enter a **BUILD COMMAND**.
+  This is the command used to deploy the application. Specify it if you want to customize the build. The default command is `npm run build`.
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md:60]: Where will the files be placed then? Logically, many people will think about it. That's why we need to clarify it here.

File: docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+3. Optional: Enter a **PUBLISH FOLDER**.
+   The deploy-ready HTML files and assets generated by the build are placed into this folder. The default folder is `/build`.
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service-guides/launching-storefronts.md:63]: Where will the files be placed then? Logically, many people will think about it. That's why we need to clarify it here.

**Comment by andriitserkovnyi** [comment]
will be merged in another PR

**Pull request closed without merging**